website/client-old/manifest.json

```diff
@@ -30,9 +30,12 @@
 
```

Comment:
Since these changes aren't going online I'd like all these new files not to be loaded, maybe you can remove them and then re-add them in the next PR?